stable/doc_hardware.md

```diff
@@ -1 +1,107 @@
-coming soon...
\ No newline at end of file
+# Hardware documentation
+
+In this section we present a short documentation of our kit. More informations about the hardware can be found in our [website](http://echopen.org/doc-website/CAT-configuration/CFG-sweeping_probe/content.html) or download our [github folder](https://github.com/echopen/PRJ-medtec_kit), aquarium configuration (PRJ-medtec_kit/configuration/CFG-aquarium_kit/readme.md). We show synoptic and block diagrams of the kit and give some informations about daughter boards and modules.
+
+### Synoptic diagram
+
+Here is a simple diagram presenting the general functions of the device.
+
+![](./doc_hardware/synoptic_diagram2.png)
+
+
+### Block diagram
```

Comment:
The bloc diagram is not up-to-date as it still shows the envelope detection, but it's not implemented on the device. 
Besides, I agree with @Bivi remark about the two diagrams. Let's get rid of this bloc diagram here, @clecoued and I will do a more general one that describes the device + app pipeline.
Then, the synoptic diagram should be more exhaustive (see first sentence in this comment)